application/utils/external_project_parsers/juiceshop.py

```diff
@@ -0,0 +1,80 @@
+import yaml
+import urllib
+from pprint import pprint
+import logging
+import os
+from typing import Dict, Any
+from application.database import db
+from application.defs import cre_defs as defs
+import re
+from application.utils import spreadsheet as sheet_utils
+from application.prompt_client import prompt_client as prompt_client
+import requests
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def parse(
+    cache: db.Node_collection,
+):
+    prompt = prompt_client.PromptHandler(cache)
+    resp = requests.get(
+        "https://raw.githubusercontent.com/juice-shop/juice-shop/master/data/static/challenges.yml"
+    )
+
+    if resp.status_code != 200:
+        logger.fatal(
+            f"could not retrieve challenges yaml, status code {resp.status_code}"
+        )
+        return
+    challenges = yaml.safe_load(resp.text)
+    for challenge in challenges:
+        chal = defs.Tool(
+            description=challenge["description"],
+            name="OWASP Juiceshop",
+            section=challenge["name"],
+            sectionID=challenge["key"],
+            hyperlink="https://preview.owasp-juice.shop/#/score-board?challenge="
```

Comment:
You are pulling the challenges from the `master` branch but then link to the environment built from `develop`. It would recommend to link to https://demo.owasp-juice.shop instead as that is consistent with `master`.